README.md

```diff
@@ -220,6 +220,15 @@ Note that:
 
 You can find the API specification [here](doc/rpc/pathfinder_rpc_api.json).
 
+### Health
+
+To check the health status of the RPC server without resorting to the [monitoring api](#monitoring-api) use the following endpoints:
+- `/` (root), provided that the request body is empty,
+- `/health`.
+
+A `200 OK` status is returned if the RPC server is healthy.
+
+Note that this method is HTTP verb agnostic.
```

Comment:
I would remove this once the `/health` is removed. Allowing `/` to return `Ok` is really only to avoid clouding the metrics with failures from bot spam.

Reply:
done